packages/driver/cypress/integration/e2e/multi-domain/snapshots/multi_domain_snapshot_aliasing.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { findCrossOriginLogs } from '../../../../support/utils'
```

Comment:
I am not a big fan of the directory name here. We aren't really testing the snapshots here (since we need cy-in-cy to likely do that well), but instead consoleProps. Any naming ideas?

Reply:
how about this?
`cypress/integration/e2e/multi-domain/consoleProps/aliasing.cy.ts`

Reply:
Refactored into the commands folder and into consoleProps suites in [bb83fd3](https://github.com/cypress-io/cypress/pull/20949/commits/bb83fd351ab399b6db2b8db65dcdac4ff6575880)